winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+
+### Formatting column without using CellsExportHandler
+
+Perform cell-level customization such as row-level styling and formatting particular columns in the exported worksheet. 
+
+In the following code sample, the NumberFormat for the `Salary` column is changed in the exported sheet after exporting without using the `CellsExportHandler`. 
+
+Reference:
+[CellRange Formatting](http://help.syncfusion.com/file-formats/xlsio/working-with-cell-or-range-formatting)
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExportMode = ExportMode.Value;
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+workBook.ActiveSheet.Columns[2].NumberFormat = "$0.0";
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-column-formatting.png" alt="Formatting the Column while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+### Alternate row styling without using CellsExportHandler
+
+In the following code sample, the background color of rows in excel is changed based on the row index using conditional formatting for better performance.
+
+Reference:
+[Conditional Formatting](http://help.syncfusion.com/file-formats/xlsio/working-with-conditional-formatting)
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExportMode = ExportMode.Value;
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+IConditionalFormats condition = workBook.ActiveSheet.Range[2, 1, this.treeGrid.View.Nodes.Count + 1, this.treeGrid.Columns.Count].ConditionalFormats;
+IConditionalFormat condition1 = condition.AddCondition();
+condition1.FormatType = ExcelCFType.Formula;
+condition1.FirstFormula = "MOD(ROW(),2)=0";
+condition1.BackColorRGB = Syncfusion.Drawing.Color.Pink;
+IConditionalFormat condition2 = condition.AddCondition();
+condition2.FormatType = ExcelCFType.Formula;
+condition2.FirstFormula = "MOD(ROW(),2)=1";
+condition2.BackColorRGB = Syncfusion.Drawing.Color.LightGray;
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-changing-row-style.png" alt="Displaying Row Style while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+## How to
+
+### Export multiple TreeGrids to single Excel sheet
+
+You can export multiple TreeGrids to single Excel sheet by merging one TreeGrid worksheet into another using the `Worksheet.UsedRange.CopyTo` method.

```

Comment:
Export multiple TreeGrids to a single Excel sheet by merging one TreeGrid worksheet into another using the `Worksheet.UsedRange.CopyTo` method.

Reply:
Updated